.github/DEVELOPMENT.md

```diff
@@ -62,3 +64,13 @@ pnpm type-check --watch
 ```
 
 You should also see suggestions from TypeScript in your editor.
+
+## Documentation
+
+You can generate documentation site with `pnpm docs`.
+To browse the docs locally you'll need a local webserver server (for VS Code users, we recommend [Live Preview](https://marketplace.visualstudio.com/items?itemName=ms-vscode.live-server)).
```

Comment:
[Question] I haven't been using one - is there a real benefit to?

[Request] If not, how about:

```suggestion
To browse the docs locally, open `docs/dev/index.html` in a web browser.
```

Reply:
You can view the pure html of `docs/*/index.html` files with the `file://` protocol, but some features that rely on JavaScript aren't available with this protocol (due to CORS policy). One being the version selector.

`docs/index.html` itself isn't viewable with the `file://` protocol.

Reply:
Ah, got it. I'd suggest adding in a `package.json` command like `"docs:serve": "http-serve docs/generated` then. But if you have a strong preference for this then I'm down to try it out 😄

Reply:
We can add that command if you like.
I was just using "Live Preview" because I already had that extension installed. You just have to right click on an html file and select "Show Preview".